FluentConstraints/FluentConstraintSet.swift

```diff
@@ -17,36 +17,36 @@ public class FluentConstraintSet {
         self.firstView = view
     }
 
-    public func build() -> [NSLayoutConstraint] {
+    open func build() -> [NSLayoutConstraint] {
         return constraints.map { $0.build() }
     }
 
-    public func activate() -> [NSLayoutConstraint] {
+    open func activate() -> [NSLayoutConstraint] {
         let constraints = build()
-        NSLayoutConstraint.activateConstraints(constraints)
+        NSLayoutConstraint.activate(constraints)
         return constraints
     }
 
     // MARK: relationship to view
 
-    public var inSuperview: FluentConstraintSet {
+    open var inSuperview: FluentConstraintSet {
         precondition(self.firstView.superview != nil, "View does not have a superview")
         self.constraints.forEach { $0.secondView = self.firstView.superview! }
         return self
     }
 
-    public func onView(view: UIView) -> FluentConstraintSet {
+    open func onView(_ view: UIView) -> FluentConstraintSet {
         self.constraints.forEach { $0.secondView = view }
         return self
     }
 
-    public func asView(view: UIView) -> FluentConstraintSet {
+    open func asView(_ view: UIView) -> FluentConstraintSet {
```

Comment:
I like this idea, but `as` is a reserved word, so the compiler won't allow me to use it as the identifier for a function. Does it make sense to update to on(view:) but keep asView(_)?